src/lib/pubkey/kyber/kyber_common/kyber.h

```diff
@@ -29,7 +29,19 @@ namespace Botan {
 
 class BOTAN_PUBLIC_API(3, 0) KyberMode {
    public:
-      enum Mode { Kyber512, Kyber512_90s, Kyber768, Kyber768_90s, Kyber1024, Kyber1024_90s };
+      enum Mode {
+         Kyber512_Round3,
+         Kyber768_Round3,
+         Kyber1024_Round3,
```

Comment:
The respective `to_string()` method and the OID-map use the short-hand "r3" already. I think, "Round3" makes it more obvious to the average user that this refers to a pre-final Kyber spec and not just some obscure technicality. But, I think, we should be consistent either way.

Reply:
Yeah good point, especially if the round3 variant ends up (as I expect) having a long life here, vs 90s mode which is probably going to be removed soonish.

Reply:
Oof - I was looking at Dilithium and there we use r3 suffix as well - and then there is a note in `dilithium.h` that we are not compatible with the round 3 submission :see_no_evil: